README.md

```diff
@@ -57,6 +57,25 @@ The `config` node holds the `YAML` that should be passed down as-is to the under
 
 At this point, the Operator does *not* validate the contents of the configuration file: if the configuration is invalid, the instance will still be created but the underlying OpenTelemetry Collector might crash.
 
+
+### Upgrades
+
+As noted above, the OpenTelemetry Collector format is continuing to evolve.  However, a best-effort attempt is made to upgrade all managed `OpenTelemetryCollector` resources.
+
+In certain scenarios, it may be desirable to prevent the operator from upgrading certain `OpenTelemetryCollector` resources. For example, when a resource is configured with a custom `.Spec.Image`, end users may wish to manage configuration themselves as opposed to having the operator upgrade it.  This can be configured on a resource by resource basis with the exposed property `.Spec.UpgradeStrategy`.
+
+By configuring a resource's `.Spec.UpgradeStrategy` to `none`, the operator will skip the given instance during the upgrade routine.
+
+The default and only other acceptable value for `.Spec.UpgradeStrategy` is `automatic`.
+
+
+### `opentelemetry-operator` vs `OpenTelemetryCollector` Versioning
```

Comment:
We have a section named "### OpenTelemetry Operator vs. OpenTelemetry Collector", perhaps it could be adapted with the extra information from this part here?

I do like the actual text, though!

Reply:
Good call, I've adjusted accordingly. Happy to tweak further if you think it still needs further clarification/etc/etc. Just let me know.